### DISCUSSION TITLE
How can I force action to execute using "repository_dispatch" event?

### DISCUSSION THREAD

**Discussion by adasq** [community/community#26122] [Actions]
<p>Hey I have a problem with triggering my actions based on custom dispatch.</p>
<p>My workflow configuration ( <strong>main.yml</strong> ):</p>
<pre><code>on:
  repository_dispatch: {}
  push:
    branches:
    - testing-gh-actions
</code></pre>
<p>I am trying to dispatch custom action with:</p>
<pre><code>curl --verbose -X POST -u "adasq:MY_TOKEN" \
-H "Accept: application/vnd.github.everest-preview+json" \
-H "Content-Type: application/json" \
https://api.github.com/repos/adasq/dropbox-v2-api/dispatches \
--data '{"event_type": "html.preview"}'
</code></pre>
<p>Do you see any problems here?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/adasq">@adasq</a>,</p>
<p>Thanks for reaching out and apologies for the delay in getting you some help. Are you receiving a webhook or other error message?</p>

**Comment by adasq** [comment]
<p>Code posted above is valid. I thought my action configuration will work on custom branch. Merging with master resolved my problem. Cheers</p>
